presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -519,16 +502,14 @@ private static String indentString(int indent)
     {
         private final Optional<StageExecutionStrategy> stageExecutionStrategy;
         private final TypeProvider types;
-        private final StatsProvider statsProvider;
-        private final CostProvider costProvider;
+        private final StatsAndCosts estimatedStatsAndCosts;
```

Comment:
`statsAndCosts`?

Reply:
there's another field called stats for execution stats.  I wanted to make the difference clear.